tests/test_Board.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+from board import Board
+from figure_factory import FigureFactory
+from figure_kind import FigureKind
+from point import Point
+
+
+class BoardTestCase(unittest.TestCase):
+
+    def test_when_figure_is_not_on_bottom(self):
+        line_figure = FigureFactory.create(FigureKind.LINE)
+
```

Comment:
please add a graphical representation of tests e.g.

Reply:
Done